src/components/Buttons/PrimaryButton/PrimaryButton.js

```diff
@@ -1,17 +1,19 @@
-import React, { Component } from "react";
+import React from "react";
 
 import "./PrimaryButton.css";
 
-class PrimaryButton extends Component {
-  render() {
-    return (
-      <div className="PrimaryButtonContainer">
-        <div className="PrimaryButton" onClick={this.props.onClick}>
-          {this.props.text}
-        </div>
+type Props = {
+   onClick: ()=>void,
+   text: string,
+}
+function PrimaryButton( props:Props ) {
```

Comment:
Since these components are literally just containers, they are fine as functional based components, but I just wanted to make sure that's what you wanted. Later as you create more complicated components and you want to control how/when they update, classes are the way to go. If you wanted to use your type Props in a class it would like:
```
class PrimaryButton extends Component<Props> {
...
}
```

Reply:
I would actually prefer if its going to have its own file, that it be a class.

Reply:
I intentionally made them functional components, since they don't need state—and never will, since they're purely presentational. I'd really like to keep them that way to prevent state from being added to them in a hacky fix for some problem down the road. And I'd like to keep them in their own file to keep them well-organized and to keep their styling separate. But if you think we should put all the buttons in the same file, we could do that too, I guess. I'm opposed to it for organizational reasons, but it wouldn't be the end of the world.

Reply:
Also, I do understand how to use prop-types in a class—if you look through the commits, you'll find that adding prop-types and switching to pure functional components were two different commits. :)